### PR TITLE
[Gekidou] update deps

### PR DESCRIPTION
**Pull request by enahum**
#### Summary
Update dependencies

```release-note
NONE
```

### CODE REVIEW

**Review by larkox — APPROVED** [review]
Only one change requested, but can be deferred for later.

File: app/database/components/index.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ type State = {
 };
 
 export function withServerDatabase<T>(Component: ComponentType<T>): ComponentType<T> {
-    return function ServerDatabaseComponent(props) {
+    return function ServerDatabaseComponent(props: any) {
```
[the remark this answers is not in the document]
Comment (enahum) [reply app/database/components/index.tsx]: done

File: types/global/managed_config.d.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-type ManagedConfig = {
+type ManagedConfig = JSX.IntrinsicAttributes & {
```
Comment (larkox) [inline types/global/managed_config.d.ts:4]: This silence the error, but the typing is wrong in emm. Fixed here: https://github.com/mattermost/react-native-emm/pull/9

Once that is fixed, this change is not needed.

Comment (enahum) [reply types/global/managed_config.d.ts]: alright, all this is now done. Thanks @larkox for the help

**Review by larkox — APPROVED** [review]

**Review by josephbaylon — COMMENTED** [review]
I've smoke tested the Android app and nothing seems to be out of the ordinary.

I'll wait for the iOS build to be fixed before I'll approve.

**Review by josephbaylon — APPROVED** [review]
LGTM. Android and iOS smoke test completed. Didn't encounter issues.

**Pull request merged**